packages/core-design-system/scripts/build.js

```diff
@@ -0,0 +1,287 @@
+     * a set in which any token could change if the theme changes.
+     * Any token that is inside such a set or is a reference with a token in that reference chain
+     * that is inside such a set, is considered "themeable",
+     * which means it could change by theme switching.
+     *
+     * This metadata is applied to the token so we can use it as a way of filtering outputs
+     * later in the "format" stage.
+     */
+    sd.registerTransform({
+      name: 'attribute/themeable',
+      type: 'attribute',
+      transform: token => {
+        function isPartOfEnabledSet(token) {
+          const set = token.filePath.replace(/^design-tokens\//g, '').replace(/.json$/g, '')
+          return themeableSets.includes(set)
+        }
+
+        // Set token to themeable if it's part of an enabled set
+        if (isPartOfEnabledSet(token)) {
+          return {
+            themeable: true
+          }
+        }
+
+        // Set token to themeable if it's using a reference and inside the reference chain
+        // any one of them is from a themeable set
+        if (usesReferences(token.original.$value)) {
+          const refs = getReferences(token.original.$value, sd.tokens)
+
+          if (refs.some(ref => isPartOfEnabledSet(ref))) {
+            return {
+              themeable: true
+            }
+          }
+        }
+      }
+    })
+
+    sd.registerTransform({
+      name: 'ts/transform/alpha',
+      type: 'value',
+      filter: prop => {
+        return (
+          prop.$extensions?.['studio.tokens']?.modify?.type === 'alpha' &&
+          prop.$extensions?.['studio.tokens']?.modify?.space === 'lch'
+        )
+      },
+      transitive: true,
+      transform: prop => {
+        const baseColor = prop.original.$value.replace(/[{}]/g, '').replace(/\./g, '-')
+        const alphaValue = prop.$extensions['studio.tokens'].modify.value
+        return `lch(from var(--${DESIGN_SYSTEM_PREFIX}-${baseColor}) l c h / ${alphaValue})`
+      }
+    })
+
+    await sd.cleanAllPlatforms()
+    await sd.buildAllPlatforms()
+  }
+
+  console.log('\n\x1b[1m\x1b[32m%s\x1b[0m', '✔︎ Generated style tokens successfully!')
+
+  // Create index.css that imports all generated files
+  await createIndexFile()
+  await createEsmIndexFile()
+
+  harnessLog()
+}
+
+async function createIndexFile() {
+  console.log(`\n\x1b[34mCreating import files in ${DESIGN_SYSTEM_ROOT}...\x1b[0m`)
+
+  // Get list of all CSS files
+  const cssFiles = (await fs.readdir(DESIGN_SYSTEM_ROOT))
+    .filter(file => file.endsWith('.css') && file !== 'index.css')
+    .sort()
+
+  // Organize files by type
+  const coreFiles = cssFiles.filter(file => !file.startsWith('dark-') && !file.startsWith('light-'))
+  const darkFiles = cssFiles.filter(file => file.includes('gitness') && file.startsWith('dark-'))
+  const lightFiles = cssFiles.filter(file => file.includes('gitness') && file.startsWith('light-'))
```

Comment:
These prefixes `dark-` and `light-` should come from a constant, for ease of maintenance in future?

Reply:
Sure, will make the change.

Reply:
Updated